src/main/scala/dal/PasswordSaltExtractor.scala

```diff
@@ -0,0 +1,13 @@
+package dal
```

Comment:
We established on Slack that the extractor wasn't required at all.

Reply:
Done